intrinsics/fnsub.go

```diff
@@ -26,6 +32,23 @@ func FnSub(name string, input interface{}, template interface{}) interface{} {
 				return src
 			}
 		}
+
+	case string:
+		// Look up references for each of the variables
+		regex := regexp.MustCompile(`\$\{([0-9A-Za-z]+)\}`)
+		variables := regex.FindAllStringSubmatch(val, -1)
+		for _, variable := range variables {
+			resolved := Ref("Ref", variable[1], template)
```

Comment:
Not so simple! 

If variable has a dot like `Foo.Arn` then it is referenced as as !GetAtt Foo.Arn. Otherwise it is !Ref